src/edu/csus/ecs/pc2/clics/API202003/EventFeedJSON.java

```diff
@@ -670,21 +662,6 @@ public long nextEventIdSequence() {
         return(++eventIdSequence);
     }
 
-    /**
-     * get event id.
-     *
-     * @param sequenceNumber
-     *            ascending number
-     * @return event Id
-     */
-    public static String getEventId(long sequenceNumber) {
-        return "pc2-" + sequenceNumber;
-    }
-
-    public static long extractSequence(String eventId) {
-        return Long.parseLong(eventId.substring(4));
-    }
-
     // TODO technical deficit - move these methods
     // TODO move pair methods into JsonUtilities
```

Comment:
Am I right that the above two "TODOs" have now been "To Done" and these comments could/should be removed?

Reply:
I don't think so?   I'm not sure what he meant by "pair methods", other than getters/setters, which appear to still be there in EventFeedJSON.java.  The same comment exists in the 2023-06 version, since it was copied from the 2020-03 version and modified.  I'm not sure why he wanted to  move those methods in the first place, since they modify / fetch things IN EventFeedJSON.

Reply:
Ok; I was just wondering.